pkg/chartutil/create.go

```diff
@@ -189,6 +190,10 @@ spec:
         app.kubernetes.io/name: {{ include "<CHARTNAME>.name" . }}
         app.kubernetes.io/instance: {{ .Release.Name }}
     spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
```

Comment:
Match the existing indentation style.

```suggestion
    {{- end }}
```